apis/addonconfigs/cni/v1alpha1/antreaconfig_types.go

```diff
@@ -258,11 +260,55 @@ type AntreaFeatureGates struct {
 
 // AntreaConfigStatus defines the observed state of AntreaConfig
 type AntreaConfigStatus struct {
+	// Message to indicate failure reason
+	// +kubebuilder:validation:Optional
+	Message string `json:"message,omitempty"`
 	// Reference to the data value secret created by controller
 	// +kubebuilder:validation:Optional
 	SecretRef string `json:"secretRef,omitempty"`
 }
 
+type AntreaNsx struct {
+	// enable indicates whether nsxt adapter shall be enabled in the cluster
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=false
+	Enable bool `json:"enable,omitempty"`
+	// bootstrapFrom either providerRef or inline configs
+	BootstrapFrom AntreaNsxBootstrapFrom `json:"bootstrapFrom,omitempty"`
+	// config is  configuration for nsxt adapter
+	AntreaNsxConfig AntreaNsxConfig `json:"config,omitempty"`
+}
+
+type AntreaNsxBootstrapFrom struct {
+	// providerRef is used with uTKG, which will be filled by uTKG Addon Controller
```

Comment:
nit: Uppercase the first letter of the comment to match the field name. applies to all the fields here